# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/xfree86/apm.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./programs/Xserver/hw/xfree86/os-support/linux/lnx_apm.c.org	Fri Apr 13 13:03:24 2001
+++ ./programs/Xserver/hw/xfree86/os-support/linux/lnx_apm.c	Fri Apr 13 13:04:06 2001
@@ -15,7 +15,7 @@
 #include <fcntl.h>
  
 #define APM_PROC   "/proc/apm"
-#define APM_DEVICE "/dev/apm_bios"
+#define APM_DEVICE "/dev/misc/apm_bios"
 
 #ifndef APM_STANDBY_FAILED
 # define APM_STANDBY_FAILED 0xf000
